src/app/contextBroker/contextBroker.cpp

```diff
@@ -466,339 +465,12 @@ static const char* validLogLevels[] =
 */
 
 
-//
-// FIXME PR-set: all these defines will be "removed" (expanded in the RestService vectors) within the #3109 refactor set of PRs
```

Comment:
Most of the FIXME PR done, but the "the RestService vectors them sleves will be moved to rest.cpp" stills, isn't it?

Reply:
Yeah, the move to another file is still pending,
But, this FIXME talks about removing the defines and putting the values inside the Rest Vectors, which is 100% finished inside this PR.
The FIXME doesn't mention the fct that we will also move the vectors.

NTC